src/services/users.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+class Users {
+  constructor() {
+    this.users = [];
+  }
+  reset() {
+    this.users = [];
+  }
+
+  getAll() {
+    return this.users;
+  }
+
+  getById(userId) {
+    const foundUser = this.users.find(user => Number(userId) === user.id);
+
+    return foundUser || null;
+  }
+
+  createUser(name) {
+    const newUser = {
+      id: Math.random(),
```

Comment:
Don't do it with `Math.random`
Create a separate function for find max id in your users, after it + 1 to this value